polynote-frontend/package.json

```diff
@@ -28,6 +28,7 @@
     "webpack-cli": "3.3.6"
   },
   "scripts": {
+    "clean": "rm dist/*.js dist/*.map || echo Nothing to clean",
```

Comment:
What is this used for?

Reply:
`npm run watch` generates tons of unused JS files over time, and they end up in the JAR. Just cleans them up.

Reply:
ah, nice. should we add it to the release process?